app/components/safe_area_view/safe_area_view.ios.js

```diff
@@ -81,13 +83,15 @@ export default class SafeAreaIos extends PureComponent {
     };
 
     getSafeAreaInsets = () => {
-        this.getStatusBarHeight();
-
-        if (this.isX) {
-            SafeArea.getSafeAreaInsetsForRootView().then((result) => {
-                const {safeAreaInsets} = result;
-                this.setState({safeAreaInsets});
-            });
+        if (this.mounted) {
```

Comment:
Maybe the `this.mounted` check should just be around the `setState` since it could get unmounted after `getSafeAreaInsetsForRootView` is called